package-lock.json

```diff
@@ -1,26 +1,26 @@
 {
   "name": "@soos-io/api-client",
-  "version": "0.2.48",
+  "version": "0.2.49-pre.1",
```

Comment:
```suggestion
  "version": "0.3.0",,
```

Reply:
(I'd just update the package json, delete the lock file and do an npm install to see if any intermediary versions get updated too)